compiler/luci-interpreter/src/kernels/Mean.cpp

```diff
@@ -245,5 +253,74 @@ void Mean::evalQuantized() const
   }
 }
 
+void Mean::evalSInt16() const
+{
+  const auto *input_data = getTensorData<int16_t>(input());
+  auto *output_data = getTensorData<int16_t>(output());
+
+  const Shape &input_shape = input()->shape();
+  const Shape &output_shape = output()->shape();
+
+  const auto *axes_data = getTensorData<int32_t>(axes());
+  const int num_axes = axes()->shape().num_elements();
+
+  constexpr int32_t output_min = -std::numeric_limits<int16_t>::max();
```

Comment:
nitpick.
If this should be particular value maybe add it explicitly?

Reply:
It's easier to make a mistake with explicit numbers.